tests/test_job.py

```diff
@@ -49,6 +51,20 @@ def test_delete_job(nomad_setup):
     test_register_job(nomad_setup)
 
 
+@pytest.mark.skipif(os.environ.get("NOMAD_VERSION") in 
```

Comment:
we can do a list comprehension into a tuple here, so we can avoid static versions and having to update these values or if a user wants to test a specific older version:

`tuple(int(i) for i in os.environ.get("NOMAD_VERSION").split(".")) > (0,5,2)`